css-positioning/absolute.css

```diff
@@ -0,0 +1,45 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  bottom: 75%;
```

Comment:
Woah, that's a big movement!! This is meant to sit in the top right, yes? Moving it from the bottom left seems like a lot of work 🤔